clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/AbstractInstanceLoadBalancerRegistrationAtomicOperation.groovy

```diff
@@ -34,8 +31,7 @@ import groovy.transform.PackageScope
 import org.springframework.beans.factory.annotation.Autowired
 
 abstract class AbstractInstanceLoadBalancerRegistrationAtomicOperation implements AtomicOperation<Void> {
```

Comment:
I'm not sure partitioning this by the type of load balancer is the correct thing given how we use it.  We typically have a specific instance and say 'add to load balancer' or 'remove from load balancer' and expect that it will fully enable or disable that instance.

With this change I think we would end up making the caller (either deck or orca) know that it has to call an add/remove from load balancer as well as an add/remove from target group